x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -213,28 +211,44 @@ export const AlertIndex = memo(() => {
               </h2>
             </EuiTitle>
           </EuiFlyoutHeader>
-          <EuiFlyoutBody />
+          <EuiFlyoutBody>
+            {selectedAlertData ? <AlertDetailsOverview /> : <EuiLoadingSpinner size="xl" />}
+          </EuiFlyoutBody>
         </EuiFlyout>
       )}
       <EuiPage data-test-subj="alertListPage" data-testid="alertListPage">
         <EuiPageBody>
           <EuiPageContent>
-            <EuiDataGrid
-              aria-label="Alert List"
-              rowCount={total}
-              columns={columns}
-              columnVisibility={useMemo(
-                () => ({
-                  visibleColumns,
-                  setVisibleColumns,
-                }),
-                [setVisibleColumns, visibleColumns]
-              )}
-              renderCellValue={renderCellValue}
-              pagination={pagination}
-              data-test-subj="alertListGrid"
-              data-testid="alertListGrid"
-            />
+            <EuiPageContentHeader>
+              <EuiPageContentHeaderSection>
+                <EuiTitle size="l">
+                  <h1>
+                    <FormattedMessage
+                      id="xpack.endpoint.alertList.viewTitle"
+                      defaultMessage="Alerts"
+                    />
+                  </h1>
+                </EuiTitle>
+              </EuiPageContentHeaderSection>
+            </EuiPageContentHeader>
+            <EuiPageContentBody>
+              <EuiDataGrid
+                aria-label="Alert List"
+                rowCount={total}
+                columns={columns}
+                columnVisibility={useMemo(
```

Comment:
for consistency - consider moving this `useMemo` to the top of the of this component